src/parse.cpp

```diff
@@ -791,6 +791,9 @@ expression* parser::parse_expression_remainder(expression* ast,
     if (!prec.math_or_logical_or_assignment) {
```

Comment:
Could we reuse `math_or_logical_or_assignment`?

Reply:
I guess we can.